dev-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3 meson
+
+DESCRIPTION="A modular Wayland compositor library"
+HOMEPAGE="https://github.com/swaywm/wlroots"
+
+EGIT_REPO_URI="https://github.com/swaywm/wlroots"
```

Comment:
This is not using a git snapshot. If we do change it, should we stop using `-9999` and use a certain revision? If so, which?

Reply:
When we update the snapshot, do we need to add `r1`, `r2` and so on?